### DISCUSSION TITLE
Android 4.2.100版本的PAGImageView 打印的日志太多了

### DISCUSSION THREAD

**Discussion by larboy1991** [Tencent/libpag#1613] [General] [answered]
【版本信息】
4.2.100 | Android | PAGViewer 版本 | AE 版本。

【平台信息】
Android 原生 

【实际的情况】
PAGImageView 打印的日志太多了，能否去除，或者是提供一个开关配置

![image](https://github.com/Tencent/libpag/assets/13670228/6a20b2b7-3795-455f-abc9-949a46fee3e1)

**Comment by peerless2012** [comment]
确实 导出使用的组件，要么不要打印调试日志，要么给个开关最好。

**Comment by kevingpqi123** [comment, marked as answer]
这里都是协助定位问题的必要 log，且是特定场景触发的，不是每帧渲染都会出现的

**Reply by kevingpqi123** [reply]
不太建议关闭类似 log
